lib/assets/javascripts/views/org_admin/templates/new.js

```diff
@@ -0,0 +1,19 @@
+import { Tinymce } from '../../../utils/tinymce';
```

Comment:
it seems that you have combined, new and edit forms so let's combine JS snippets into one file too